Automattic/vip-go-mu-plugins
# Updated wording on VIP Page Purge functionality

## Description
Should always be represented as "Purge", previously it was inconsistently using the word Flush synonymously with purge

## Changelog Description
### Page Cache Purge will always be referred to as "Purge"
Previously the mu-plugins admin bar was inconsistently referring to this as a flush. We changed the wording to "Purge" in all circumstances so that our users will not be confused by using flush as a synonym.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Log into the admin
1. View any post or page
1. Verify the button on the admin bar says "Purge Cache for Page" and not Flush.
2. Click the button and Verify the functionality works (The event handler has been changed from being called flush to purge)

## Screenshots
![Screen Shot 2022-06-27 at 11 11 03 AM](https://user-images.githubusercontent.com/46167019/175997757-81ad422c-80f7-4ba2-b5d6-bbd41a0ed887.png)
![Screen Shot 2022-06-27 at 11 11 13 AM](https://user-images.githubusercontent.com/46167019/175997769-4d566bdb-2040-43a8-8e9b-4b567f0bae27.png)
